bin/journey-test.sh

```diff
@@ -233,15 +233,15 @@ solve_all_exercises() {
 
     pushd ${exercism_exercises_dir}/${TRACK}/${exercise}
     # Check that tests compile before we strip @Ignore annotations
-    "$EXECPATH"/gradlew compileTestJava
+    "$EXECPATH"/gradlew compileTestJava --configure-on-demand --parallel
```

Comment:
I re ran them again and it didn't seem to make much of a change. I guess something might have been up with that first build. Should I remove the --configure-on-demand tags?